### PR TITLE
Started writing the discussion

### PR DESCRIPTION
**Pull request by jokingrottten**
Addressed:
-what we did in the lab
-some error
-why it's a "phase-lag circuit"

### CODE REVIEW

**Comment by jokingrottten** [comment]
Wow, it worked!

File: lab02/sections/discussion.tex [hunk 1]
```diff
@@ -1 +1,14 @@
 \section{Discussion}\label{sec:discussion}
+A phase-lag circuit was built of an identical topology to that in Figure 1.
+The frequency of a 1V peak-to-peak input signal was varied over ~12kHz in 1kHz increments. 
+The input and output voltage and phase were measured using the in-lab oscilloscope.
+The recorded data can be seen in Table 1.
+
+In our pre-lab, we simulated the phase and amplitude response of the phase-lag system.
+As can be seen by comparing Figures XXX and XXY, the simulated results closely matched experimentally generated values.
+Both phase plots show a minimum phase of -55\si{\degree} occurring at ~3kHz. 
+The capacitors used had an error of $\pm$20$\%$.
```
Comment (trstephen) [inline lab02/sections/discussion.tex:10]: Escaping `20` from math mode isn't necessary, but it prints the same :stuck_out_tongue:

File: lab02/sections/discussion.tex [hunk 2]
```diff
@@ -1 +1,14 @@
 \section{Discussion}\label{sec:discussion}
+A phase-lag circuit was built of an identical topology to that in Figure 1.
+The frequency of a 1V peak-to-peak input signal was varied over ~12kHz in 1kHz increments.
```
Comment (trstephen) [inline lab02/sections/discussion.tex:3]: Welcome to Latex weirdness! 

Things that you'd think give you ~ but don't:
- `~` is a special character related to character spacing
- `\~` puts a tilde over a character.
- `\textasciitilde` will _almost_ get you what you want, but it's, uh, too high

Things that actually work:
- `$\sim$`, but it's pretty big
- `$\approx$` is what I use!

I'd write this as `$\approx \SI{12}{\kilo\hertz}$ in 1 kHz increments.` What's the `\SI` for? It allows units to appear non-italic and properly spaced in math mode (try `$2 hz$` vs `$2 \si{\hertz}$`). You can use it as `\SI{number}{\units}` or `\si{units}`. 
Honestly, it's overkill a lot of the time, like when you're just referencing values in a discussion but it's really useful when you start mixing equations and values.

File: lab02/sections/discussion.tex [hunk 3]
```diff
@@ -1 +1,14 @@
 \section{Discussion}\label{sec:discussion}
+A phase-lag circuit was built of an identical topology to that in Figure 1.
```
Comment (trstephen) [inline lab02/sections/discussion.tex:2]: You can insert dynamic references by using `\ref{name}` where `name` corresponds to the `\label{name}` field in the figure, section, table, whatever you're referencing.

Comment (trstephen) [inline lab02/sections/discussion.tex:2]: "... was built ~~of~~ from an identical ..."

File: lab02/sections/discussion.tex [hunk 4]
```diff
@@ -1 +1,14 @@
...
+The frequency of a 1V peak-to-peak input signal was varied over ~12kHz in 1kHz increments. 
+The input and output voltage and phase were measured using the in-lab oscilloscope.
+The recorded data can be seen in Table 1.
```
Comment (trstephen) [inline lab02/sections/discussion.tex:5]: You can use a dynamic table reference.

**Comment by mtayler** [comment]
I'm pretty sure I rebased this correctly, but... Git is hard

EDIT: Okay, so rebasing doesn't work for this because it's already been pushed...

EDIT2: Oh god what have I done

**Comment by mtayler** [comment]
Okay I think I got it figured out. Opened a new discussion branch rebased onto master.

**Pull request closed without merging**